base/python/R/R_nonstandard/rstudio_shiny/shiny-server-standalone.conf

```diff
@@ -0,0 +1,28 @@
+# Define the user we should use when spawning R Shiny processes
```

Comment:
This is useful, but should rather be in its own image, depending on `R_nonstandard`. Please park in another branch until we do that.

Reply:
Done!